README.md

```diff
@@ -339,14 +339,20 @@ A number of options are available to allow you to customise the SDK:
   ### document ###
 
   This is the document capture step. Users will be asked to select the document type and to provide images of their selected documents. They will also have a chance to check the quality of the images before confirming.
+
   The custom options are:
-  - documentTypes
+  - documentTypes (object)
+
+  The list of document types can be filtered by using the `documentTypes` option. The default value for each document type is `true`.
```

Comment:
I would add "... document types _visible to the user_ can be"

Reply:
this can make it clear that it will change UI

Reply:
good point